modules/buttons/LINKEDIN_BUTTON_BY_Rakesh-116.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <style>
+      .Linkedin-button{
+        background-color:#0072b1;
+        padding:10px;
+        width:120px;
+        border-width: 0;
+        border-radius:2px;
+        color:#ffffff;
+        font-size:15px;
+        font-weight: 100;
+      }
+    </style>
+    <div>
+        <button class="Linkedin-button">
+            <span style="font-weight: 600;font-size:20px; padding:5px;">in</span>Linkedin
```

Comment:
I like this approach. Funny and brief. No svgs not images.... LOL

Reply:
Ha, as u said less memory needed